.storybook/preview.ts

```
@@ -0,0 +1,22 @@
+import type { Preview } from '@storybook/nextjs-vite';
+import '@/style/global.css';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)